frontend/src/app/cache.ts

```diff
@@ -187,6 +215,19 @@ export const cacheSlice = createSlice({
 
         state.instructorResults[action.meta.arg] = action.payload;
       });
+
+    builder
+      .addCase(fetchAllInstructors.pending, (state) => {
+        state.instructorsLoading = true;
+      })
+      .addCase(fetchAllInstructors.fulfilled, (state, action) => {
+        state.instructorsLoading = false;
+        if (action.payload) {
+          state.allInstructors = action.payload;
+          state.fuseIndex = Fuse.createIndex(["instructor"], action.payload).toJSON();
```

Comment:
I'm not too sure on the `Fuse` library API so I will trust this here

Reply:
Was basing it of this https://www.fusejs.io/api/indexing.html#fuse-createindex